--- a/O.meson.build	2020-04-02 09:42:32.615382574 +0200
+++ b/meson.build	2020-04-02 09:42:47.861838761 +0200
@@ -148,7 +148,7 @@
 systemsleepdir = join_paths(rootlibexecdir, 'system-sleep')
 systemunitdir = join_paths(rootprefixdir, 'lib/systemd/system')
 systempresetdir = join_paths(rootprefixdir, 'lib/systemd/system-preset')
-udevlibexecdir = join_paths(rootprefixdir, 'lib/udev')
+udevlibexecdir = '/ro/lib/udev'
 udevrulesdir = join_paths(udevlibexecdir, 'rules.d')
 udevhwdbdir = join_paths(udevlibexecdir, 'hwdb.d')
 catalogdir = join_paths(prefixdir, 'lib/systemd/catalog')
