From faa7a393f60370d03e863253e0ba80f4fbcdb14f Mon Sep 17 00:00:00 2001
From: Michael Stapelberg <michael@stapelberg.de>
Date: Wed, 26 Dec 2018 22:48:39 +0100
Subject: [PATCH] fix path

---
 configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure b/configure
index f4e9c72..7476dbb 100755
--- a/configure
+++ b/configure
@@ -15339,8 +15339,7 @@ rm -f core conftest.err conftest.$ac_objext \
 
 GLIB_CFLAGS="$GLIB_CFLAGS -DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_46"
 
-GIO_MODULE_DIR=$($PKG_CONFIG --variable giomoduledir gio-2.0)
+GIO_MODULE_DIR=/ro/glib-networking-amd64-2.54.1-1/out/lib/gio
 if test "$GIO_MODULE_DIR" = ""; then :
   { { $as_echo "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
 $as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
-- 
2.19.1

