--- old/template/ruby.pc.in	2020-03-31 12:44:55.000000000 +0200
+++ new/template/ruby.pc.in	2020-05-02 13:24:09.239957802 +0200
@@ -48,12 +48,11 @@
 LIBRUBYARG_STATIC=@LIBRUBYARG_STATIC@
 LIBRUBYARG=@LIBRUBYARG@
 LIBS=@LIBS@
-DLDFLAGS=@DLDFLAGS@
 
 Name: Ruby
 Description: Object Oriented Script Language
 Version: ${ruby_version}
 URL: https://www.ruby-lang.org
 Cflags: -I${rubyarchhdrdir} -I${rubyhdrdir}
-Libs: ${DLDFLAGS} ${LIBRUBYARG_SHARED} ${LIBS}
+Libs: ${LIBRUBYARG_SHARED} ${LIBS}
 Requires:
Binary files /dev/null and new/template/ruby.pc.in~ differ
