diff --git i/modules.d/90dm/59-persistent-storage-dm.rules w/modules.d/90dm/59-persistent-storage-dm.rules
index 73b09376..3eff84ac 100644
--- i/modules.d/90dm/59-persistent-storage-dm.rules
+++ w/modules.d/90dm/59-persistent-storage-dm.rules
@@ -3,9 +3,14 @@ ACTION!="add|change", GOTO="dm_end"
 # Also don't process disks that are slated to be a multipath device
 ENV{DM_MULTIPATH_DEVICE_PATH}=="?*", GOTO="dm_end"
 
+IMPORT{program}="/sbin/dmsetup info -c --nameprefixes --unquoted --rows --noheadings -o name,uuid,suspended,readonly,major,minor,open,tables_loaded,names_using_dev -j%M -m%m"
+
+ENV{DM_NAME}=="?*", SYMLINK+="mapper/$env{DM_NAME}"
+SYMLINK+="disk/by-id/dm-name-$env{DM_NAME}"
+ENV{DM_UUID}=="?*", SYMLINK+="disk/by-id/dm-uuid-$env{DM_UUID}"
+
 KERNEL!="dm-[0-9]*", GOTO="dm_end"
 ACTION=="add", GOTO="dm_end"
-IMPORT{program}="/sbin/dmsetup info -c --nameprefixes --unquoted --rows --noheadings -o name,uuid,suspended,readonly,major,minor,open,tables_loaded,names_using_dev -j%M -m%m"
 ENV{DM_NAME}!="?*", GOTO="dm_end"
 ENV{DM_UDEV_DISABLE_OTHER_RULES_FLAG}=="1", GOTO="dm_end"
 ENV{DM_UUID}=="CRYPT-TEMP-?*", GOTO="dm_end"
