--- old/src/loadkeys.c	2017-01-08 19:36:15.000000000 +0100
+++ new/src/loadkeys.c	2019-06-29 23:35:18.089546066 +0200
@@ -27,7 +27,7 @@
 #include "keymap.h"
 
 static const char *progname         = NULL;
-static const char *const dirpath1[] = { "", DATADIR "/" KEYMAPDIR "/**", KERNDIR "/", 0 };
+static const char *const dirpath1[] = { "", "/ro/share/keymaps" "/**", KERNDIR "/", 0 };
 static const char *const suffixes[] = { "", ".kmap", ".map", 0 };
 
 static void __attribute__((noreturn))
--- old/src/setfont.c	2017-01-08 19:36:15.000000000 +0100
+++ new/src/setfont.c	2019-06-29 23:35:21.797495504 +0200
@@ -53,10 +53,10 @@
 int debug   = 0;
 
 /* search for the font in these directories (with trailing /) */
-const char *const fontdirpath[]  = { "", DATADIR "/" FONTDIR "/", 0 };
+const char *const fontdirpath[]  = { "", "/ro/share/consolefonts" "/", 0 };
 const char *const fontsuffixes[] = { "", ".psfu", ".psf", ".cp", ".fnt", 0 };
 /* hide partial fonts a bit - loading a single one is a bad idea */
-const char *const partfontdirpath[]  = { "", DATADIR "/" FONTDIR "/" PARTIALDIR "/", 0 };
+const char *const partfontdirpath[]  = { "", "/ro/share/consolefonts" "/" PARTIALDIR "/", 0 };
 const char *const partfontsuffixes[] = { "", 0 };
 
 static inline int
