--- old/tools/l2test.c	2016-10-28 14:57:27.000000000 +0200
+++ new/tools/l2test.c	2019-09-14 18:11:06.048024360 +0200
@@ -906,7 +906,7 @@
 			if (timestamp) {
 				struct timeval tv;
 
-				if (ioctl(sk, SIOCGSTAMP, &tv) < 0) {
+				if (ioctl(sk, SIOCGSTAMP_OLD, &tv) < 0) {
 					timestamp = 0;
 					memset(ts, 0, sizeof(ts));
 				} else {
--- old/tools/rctest.c	2015-09-04 03:19:36.000000000 +0200
+++ new/tools/rctest.c	2019-09-14 18:11:06.060024102 +0200
@@ -504,7 +504,7 @@
 			if (timestamp) {
 				struct timeval tv;
 
-				if (ioctl(sk, SIOCGSTAMP, &tv) < 0) {
+				if (ioctl(sk, SIOCGSTAMP_OLD, &tv) < 0) {
 					timestamp = 0;
 					memset(ts, 0, sizeof(ts));
 				} else {
