diff --git i/elf/ldd.bash.in w/elf/ldd.bash.in
index 14f9787..048faab 100644
--- i/elf/ldd.bash.in
+++ w/elf/ldd.bash.in
@@ -26,7 +26,7 @@
 TEXTDOMAIN=libc
 TEXTDOMAINDIR=@TEXTDOMAINDIR@
 
-RTLDLIST=@RTLD@
+RTLDLIST=/ro/glibc-i686-amd64-2.31-7/out/lib/ld-linux.so.2
 warn=
 bind_now=
 verbose=
diff --git i/sysdeps/unix/sysv/linux/x86_64/ldd-rewrite.sed w/sysdeps/unix/sysv/linux/x86_64/ldd-rewrite.sed
index 44d76e8..f0a192c 100644
--- i/sysdeps/unix/sysv/linux/x86_64/ldd-rewrite.sed
+++ w/sysdeps/unix/sysv/linux/x86_64/ldd-rewrite.sed
@@ -1,3 +1,2 @@
 /LD_TRACE_LOADED_OBJECTS=1/a\
 add_env="$add_env LD_LIBRARY_VERSION=\\$verify_out"
-s_^\(RTLDLIST=\)\(.*lib\)\(\|64\|x32\)\(/[^/]*\)\(-x86-64\|-x32\)\(\.so\.[0-9.]*\)[ 	]*$_\1"\2\4\6 \264\4-x86-64\6 \2x32\4-x32\6"_
