--- old/data/lightdm.conf	2020-04-25 09:25:55.360171363 +0200
+++ new/data/lightdm.conf	2020-04-25 09:26:06.350020384 +0200
@@ -88,7 +88,7 @@
 #pam-service=lightdm
 #pam-autologin-service=lightdm-autologin
 #pam-greeter-service=lightdm-greeter
-#xserver-command=X
+xserver-command=Xorg
 #xmir-command=Xmir
 #xserver-config=
 #xserver-layout=
@@ -99,7 +99,7 @@
 #xdmcp-manager=
 #xdmcp-port=177
 #xdmcp-key=
-#greeter-session=example-gtk-gnome
+greeter-session=lightdm-gtk-greeter
 #greeter-hide-users=false
 #greeter-allow-guest=true
 #greeter-show-manual-login=false
